src/Arcus.WebApi.Unit/Arcus.WebApi.Unit.csproj

```diff
@@ -1,34 +1,37 @@
-        <Description>Provides capabilities to easily build Web APIs running in Azure.</Description>
-        <Copyright>Copyright (c) Arcus</Copyright>
-        <PackageLicenseUrl>https://github.com/arcus-azure/arcus.webapi/blob/master/LICENSE</PackageLicenseUrl>
-        <PackageProjectUrl>https://github.com/arcus-azure/arcus.webapi</PackageProjectUrl>
-        <RepositoryUrl>https://github.com/arcus-azure/arcus.webapi</RepositoryUrl>
-        <PackageIconUrl>https://raw.githubusercontent.com/arcus-azure/arcus/master/media/arcus.png</PackageIconUrl>
-        <GenerateDocumentationFile>true</GenerateDocumentationFile>
-        <DocumentationFile>Arcus.WebApi.Unit.Open-Api.xml</DocumentationFile>
-    </PropertyGroup>
-    <ItemGroup>
-      <PackageReference Include="BouncyCastle" Version="1.8.5" />
-      <PackageReference Include="BouncyCastle.NetCoreSdk" Version="1.9.0.1" />
-      <PackageReference Include="System.Web.Http" Version="4.0.0" />
-      <PackageReference Include="Microsoft.AspNetCore.App" />
-      <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.2.0" />
-      <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="2.2.0" />
-      <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.3.0" />
-      <PackageReference Include="Microsoft.OpenApi.Readers" Version="1.1.4" />
-      <PackageReference Include="xunit" Version="2.4.1" />
-      <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
-    </ItemGroup>
-    <ItemGroup>
-      <ProjectReference Include="..\Arcus.WebApi.OpenApi.Extensions\Arcus.WebApi.OpenApi.Extensions.csproj" />
-      <ProjectReference Include="..\Arcus.WebApi.Security.Authentication\Arcus.WebApi.Security.Authentication.csproj" />
-      <ProjectReference Include="..\Arcus.WebApi.Security\Arcus.WebApi.Security.csproj" />
-    </ItemGroup>
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.2;netcoreapp3.0</TargetFrameworks>
+    <Authors>Arcus</Authors>
+    <Company>Arcus</Company>
+    <RepositoryType>Git</RepositoryType>
+    <PackageTags>Azure;WebAPI;App Services;Web App;Web;API</PackageTags>
+    <Description>Provides capabilities to easily build Web APIs running in Azure.</Description>
+    <Copyright>Copyright (c) Arcus</Copyright>
+    <PackageLicenseUrl>https://github.com/arcus-azure/arcus.webapi/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/arcus-azure/arcus.webapi</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/arcus-azure/arcus.webapi</RepositoryUrl>
+    <PackageIconUrl>https://raw.githubusercontent.com/arcus-azure/arcus/master/media/arcus.png</PackageIconUrl>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <OpenApiGenerateDocuments>false</OpenApiGenerateDocuments>
+    <DocumentationFile>Arcus.WebApi.Unit.Open-Api.xml</DocumentationFile>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="BouncyCastle" Version="1.8.5" />
```

Comment:
Do we have to list them all here? Normally framework does this for us based on the dependencies, no?

Reply:
You mean the conditions on the package references? Yeah, I had to add those to make it fully compatible for both.